src/lib/snarky_js_bindings/test_module/simple-zkapp-mock-apply.js

```diff
@@ -90,6 +91,16 @@ partiesJsonInitialize = signFeePayer(partiesJsonInitialize, senderKey, {
 });
 toc();
 
+// verify the proof
+tic("verify transaction proof");
```

Comment:
Nit: `tic` and `tock` would be better served by self-documenting names. Consider e.g. `startTiming` and `endTiming`.
(Not to change in this PR, it's already a grab-bag, lets not make it worse.)